.github/workflows/phase_1_python.yml

```diff
@@ -0,0 +1,210 @@
+---
+name: Phase 1 - Python
+on: [push]
+
+env:
+  TRIVY_VERSION: 0.54.1
+  SBOMQS_VERSION: 0.1.7
```

Comment:
@vpetersson can we update this to v0.1.9, there was a bug in computing the license counts in this version